proto/spire/api/agent/delegation/v1/delegation.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+package spire.api.agent.delegation.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/agent/delegation/v1;delegationv1";
+
+import "spire/api/types/selector.proto";
+import "spire/api/types/x509svid.proto";
+
+// The deletation service provides an interface to get the SVIDs of other
+// workloads on the host. This service is intended for use cases where a process
+// (different than the workload one) should access the workload's SVID to
+// perform actions on behalf of the workload. One example of is using a single
+// node instance of Envoy that upgrades TCP connections for different processes
+// running in such a node.
+//
+// The caller must be local and its identity must be listed in the allowed
+// clients on the spire-agent configuration.
+service Delegation {
+    // Fetch X.509-SVIDs for multiple workloads.
+    rpc FetchX509SVIDs(stream FetchX509SVIDsRequest) returns (stream FetchX509SVIDsResponse);
```

Comment:
Since have a significant departure from the workload API here, perhaps a different RPC name makes sense?

To me, this RPC has a sessionized feel to it.

`SubscribeToX509SVIDs`?
`StartSessionX509SVID`?
`ReceiveX509SVIDs`?
`AttachX509SVIDStream`?

The bundle RPC would be updated as well to match e.g. `AttachBundleStream`

Reply:
Out of those, I'm leaning towards SubscribeToX509SVIDs the most I think?